src/overlays/actors/ovl_Bg_Spot15_Rrbox/z_bg_spot15_rrbox.c

```diff
@@ -9,7 +9,25 @@ void BgSpot15Rrbox_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgSpot15Rrbox_Update(Actor* thisx, GlobalContext* globalCtx);
 void BgSpot15Rrbox_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void func_808B4084(BgSpot15Rrbox* this, GlobalContext* globalCtx);
+void func_808B40AC(BgSpot15Rrbox* this, GlobalContext* globalCtx);
+void func_808B4178(BgSpot15Rrbox* this, GlobalContext* globalCtx);
+void func_808B4194(BgSpot15Rrbox* this, GlobalContext* globalCtx);
+void func_808B4380(BgSpot15Rrbox* this, GlobalContext* globalCtx);
+void func_808B43D0(BgSpot15Rrbox* this, GlobalContext* globalCtx);
+void func_808B44B8(BgSpot15Rrbox* this, GlobalContext* globalCtx);
+void func_808B4178(BgSpot15Rrbox* this, GlobalContext* globalCtx);
+void func_808B44CC(BgSpot15Rrbox* this, GlobalContext* globalCtx);
+
+// Prototypes for functions external to this file
+f32 func_8003CA64(CollisionContext* colCtx, CollisionPoly** outPoly, s32* bgId, Actor* actor, Vec3f* pos, f32 chkDist);
+s32 func_800435D8(GlobalContext* globalCtx, DynaPolyActor* dyna, s16 arg2, s16 arg3, s16 arg4);
```

Comment:
pretty sure "external" function prototypes should go into functions.h

Reply:
Moved them there.